packages/@okta/vuepress-site/docs/guides/terraform-configure-device-signin-standards/main/index.md

```diff
@@ -0,0 +1,383 @@
+
+1. Add the `okta_policy_device_assurance_ios` resource to your configuration.
+
+1. Set its `name` argument to a user-visible name for this policy.
+
+1. Set arguments for different signals. The following are the most commonly checked signals. For the complete list of signals, see the [Terraform provider documentation for this resource](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/policy_device_assurance_ios)
+
+    * `os_version`: The minimum operating system for a device.
+
+    * `jailbreak`: Specifies if jailbreak devices are allowed. To forbid jailbroken devices, set this to `false` and use it in an authentication rule that checks for this device assurance policy. Set the rule to **allow** authentication. Setting it to `true` means that this device assurance policy requires jailbreaking, which you could use with an authentication rule that explicitly **denies** access.
+
+    * `screenlock_type`: A set of required screen lock actions and behaviors. The supported options are `BIOMETRIC` (biometric locking) and `PASSCODE` (passcode locking).
+
+The following code shows a typical iOS device assurance policy:
+
+```hcl
+resource okta_policy_device_assurance_ios ios_example {
+  name = "iOS example"
+  os_version = "12.4.5"
+  jailbreak = false
+  screenlock_type = toset(["BIOMETRIC"])
+}
+```
+
+### macOS
+
+Create a macOS device assurance policy:
+
+1. Add the `okta_policy_device_assurance_macos` resource to your configuration.
+
+1. Set the `name` argument to a user-visible name for this policy.
+
+1. Set arguments for different signals. The following are the most commonly checked signals. For the complete list of signals, see the [Terraform provider documentation for this resource](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/policy_device_assurance_macos).
+
+    * `os_version`: The minimum operating system version for a device.
+
+    * `disk_encryption_type`: The set of required encryption settings for the device. The only option is `ALL_INTERNAL_VOLUMES` ( all internal volumes are encrypted).
+
+    * `screenlock_type`: A set of required screen lock actions and behaviors.  The supported options are `BIOMETRIC` (biometric locking) and `PASSCODE` (passcode locking).
+
+Your policy can also check signals on devices from a supported third-party signal provider, such as the Google Chrome Device Trust Connector in the Chrome browser. Specify its device signals in your Terraform code with the fields with the prefix `tpsp_`. For example, `tpsp_os_version` instead of `os_version`.  Okta for device assurance for ChromeOS requires using [Workforce Identity Adaptive Multi-factor Authentication (AMFA)](https://www.okta.com/learn/adaptive-mfa/). Developer Edition Okta orgs don't support the Google Chrome Device Trust Connector by default. If you're a paying customer, contact Okta developer support to enable third-party signal support on a Developer Edition Okta org.
```

Comment:
```suggestion
Your policy can also check signals on devices from a supported third-party signal provider, such as the Google Chrome Device Trust Connector in the Chrome browser. Specify its device signals in your Terraform code with the fields with the prefix `tpsp_`. For example, `tpsp_os_version` instead of `os_version`.  Okta for device assurance for ChromeOS requires using [Workforce Identity Adaptive Multi-factor Authentication (AMFA)](https://www.okta.com/learn/adaptive-mfa/). Developer Edition orgs don't support the Google Chrome Device Trust Connector by default. If you're a paying customer, contact Okta developer support to enable third-party signal support on a Developer Edition org.
```